android-core/src/androidTest/kotlin/com.mparticle/SessionMessagesTest.kt

```diff
@@ -28,10 +28,10 @@ class SessionMessagesTest : BaseCleanStartedEachTest() {
     fun testSessionStartMessage() {
         val sessionStartReceived = BooleanArray(1)
         sessionStartReceived[0] = false
-        Assert.assertFalse(mAppStateManager.session.isActive)
+        Assert.assertFalse(mAppStateManager.fetchSession().isActive)
```

Comment:
Why did this change to `fetchSession()`? I don't see that in the code changes.

Reply:
so with same name it's kotlin compiler issue, so It was giving compiling error for conflict . Hence I changed the method name.